templates/zerver/app/index.html

```diff
@@ -92,7 +91,12 @@ <h3>{{ _('Loading...') }}</h3>
         <div class="column-left">
             {% include "zerver/app/left_sidebar.html" %}
         </div>
-        <div class="column-middle">
+        <div id="recent_topics_view">
+            <div class="recent_topics_container">
+                <div id="recent_topics_table"></div>
+            </div>
+        </div>
```

Comment:
Structurally, I'd expect this to be inside `column-middle-inner` and `<div class="tab-pane active" id="home">` to have the `home `renamed to `message_feed_container` (your `messages_view`), in this template's hierarchy.